docs/0.2/creating_products.md

```diff
@@ -1,22 +1,25 @@
 # Creating Products with Grid on Splinter
 
 <!--
-  Copyright (c) 2018-2020 Cargill Incorporated
+  Copyright (c) 2018-2021 Cargill Incorporated
   Licensed under Creative Commons Attribution 4.0 International License
   https://creativecommons.org/licenses/by/4.0/
 -->
 
 This procedure explains how to create and manage Grid products
-(trade item data) using Grid's command-line interface.
+(trade item data) using Grid's command-line interface. Grid products consist of
```

Comment:
This should explain that the default schema supports GDSN products, but users can define their own schema that can leverage GDSN along with other attributes, or another schema entirely. I think the main section of this guide should cover creating a product with just the default schema, and then add a section at the end which describes the "extended attributes" pattern, or using a different schema. Since the power of Grid is to align on open standards, patterns other than this default should be treated as an edge case.